app/src/main/java/com/practicum/playlistmaker/ui/search/fragment/SearchFragment.kt

```diff
@@ -102,7 +111,7 @@ class SearchActivity : AppCompatActivity() {
             viewModel.stopSearch()
             binding.queryInput.setText("")
             viewModel.savedSearchRequest = ""
-            val keyboard = getSystemService(INPUT_METHOD_SERVICE) as InputMethodManager
+            val keyboard = requireActivity().getSystemService(AppCompatActivity.INPUT_METHOD_SERVICE) as InputMethodManager
```

Comment:
🍏 Здесь все действия можно производить с помощью ближайшего доступного контекста:
```suggestion
            val keyboard = requireContext().getSystemService(INPUT_METHOD_SERVICE) as InputMethodManager
```